vNext/Templates/OrleansInterface/.template.config/template.json

```diff
@@ -0,0 +1,35 @@
+{
+    "author": "Microsoft",
+    "classification": [
+        "Orleans",
+        "Cloud-Computing",
+        "Actor-Model",
+        "Actors",
+        "Distributed-Systems",
+        "C#",
+        "Silo",
+        ".NET"
+    ],
+    "name": "Microsoft Orleans (2.0 Technical Preview) Interface",
+    "identity": "Microsoft.Orleans.Templates.Interface",
+    "shortname": "orleans-interface",
+    "tags": {
+        "language": "C#",
+        "type": "project"
+    },
+    "sourceName": "OrleansInterface",
+    "preferNameDirectory": "true",
+    "postActions": [
+        {
+            "condition": "(!skipRestore)",
+            "description": "Restore Orleans NuGet packages required by this project.",
+            "manualInstructions": [
+                {
+                    "text": "Run 'dotnet restore'"
```

Comment:
I suppose this can be removed with the latest .net core cli since this is supposed to be implicit now, right?